dashboard/client/src/components/ActorTable.tsx

```diff
@@ -59,8 +98,19 @@ const ActorTable = ({
   const [actorIdFilterValue, setActorIdFilterValue] = useState(filterToActorId);
   const [pageSize, setPageSize] = useState(10);
   const { ipLogMap } = useContext(GlobalContext);
+
   const actorList = Object.values(actors || {}).filter(filterFunc);
```

Comment:
Since this gets created every single render, seems like we will re-render the entire table many times. Can we put this in the useMemo as well?

I think we should depend on the `actors` prop to decide if we need to update the memo. Can we also check all the classes that are using this component to see if this actors prop only changes when the underlying value changes?